src/main/java/bio/terra/cli/serialization/userfacing/input/CreateBqDatasetParams.java

```diff
@@ -15,12 +15,16 @@
   public final String projectId;
   public final String datasetId;
   public final String location;
+  public final Integer defaultPartitionLifetime;
+  public final Integer defaultTableLifetime;
```

Comment:
Integer time durations make me unhappy, but if we're really just using them in the serialization format, I guess it's ok. We could look at jackson java8 time support, but that's probably not helpful to users...
Let's at least add meaningful time unit suffixes to this and the other classes.

Reply:
I think it's also relevant that the `bq` CLI uses integer seconds for these two properties. If there were no precedent here, I'd probably choose to use days or Duration, but my preference is to mimic the existing cloud CLI where applicable. This is also similar to how we approached the GCS bucket lifecycle rules (i.e. use exact same JSON format as the `gsutil` CLI).